packages/webrtc-core/src/index.ts

```diff
@@ -2,11 +2,15 @@
 // support 'export * as ___ from ___' yet.
 import * as media from './media';
 
+// TODO: webex js sdk re export the types for device-manager
+
 export * from './device/device-management';
 export * from './media/local-camera-track';
 export * from './media/local-display-track';
+export * from './media/local-computer-audio-track';
 export * from './media/local-microphone-track';
 export * from './media/local-track';
+export * from './media/local-video-track';
```

Comment:
I think you're missing `export * from './media/local-audio-track';` from here as well.

Reply:
will check and add it

Reply:
fixed it